utils/src/main/scala/cromwell/pipeline/utils/json/AdtJsonFormatter.scala

```diff
@@ -0,0 +1,95 @@
+package cromwell.pipeline.utils.json
+
+import cromwell.pipeline.utils.json.AdtJsonFormatter._
+import play.api.libs.json._
+
+import scala.reflect.{ClassTag, classTag}
```

Comment:
Formatter should left space between brackets and letters

Reply:
Ooops. Fixed.